.github/workflows/docs-publish.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - release-2.x
+      - release-3.x
```

Comment:
I think what this does is that it launches  a netlify build on pushes to 3.x so we don't have to wait for something to change on main. And the netlify build checks out `release-3.x` alongside `main` and `release-2.x`. It's good to have it in all cases 👍